src/core/lib/security/credentials/tls/grpc_tls_certificate_distributor.h

```diff
@@ -213,4 +212,6 @@ struct grpc_tls_certificate_distributor
       ABSL_GUARDED_BY(mu_);
 };
 
+typedef TlsCertificateDistributor TlsCertificateDistributor;
```

Comment:
Is this necessary?

Reply:
Ah good catch, that was a temporary typedef I put in for the old `grpc_tls_certificate_distributor` name

Then at some point it got changed in a full find and replace - removed